chitchat_app/lib/main.dart

```diff
@@ -23,7 +25,7 @@ class MyApp extends StatelessWidget {
         colorScheme: ColorScheme.fromSeed(seedColor: Colors.orange), // Sets color theme.
         useMaterial3: true,
       ),
-      home: const MainPage(), // Main page of the app.
+      home: const MainPage(), // Main page of the app, now not a constant.
```

Comment:
This says now not a constant but is still marked `const`. I assume this is for the flutter lint but please double check if this is supposed to be const or not.